users/views.py

```diff
@@ -215,3 +218,43 @@ def post(self, request, uidb64, token):
             )
         else:
             return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+
+class UserProfileView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    def get_current_user(self, request):
+        jwt_token = request.auth
+        user_id = jwt_token.payload.get("id")
+        user_instance = CustomUser.objects.get(id=user_id)
+        return user_instance
+
+    def get(self, request):
+        user_instance = self.get_current_user(request)
+        serializer = UserSerializer(user_instance)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    def put(self, request):
+        user_instance = self.get_current_user(request)
+        serializer = UserSerializer(instance=user_instance, data=request.data)
+
+        if serializer.is_valid():
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_200_OK)
+        else:
+            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+    def delete(self, request):
+        user_instance = self.get_current_user(request)
+
+        if user_instance.is_investor == 1:
```

Comment:
We can have only one startup, but there can be several investors. Please make the appropriate changes.

Reply:
We have one startup and one investor per user